src/it/scala/org/ergoplatform/it/Docker.scala

```diff
@@ -79,8 +79,10 @@ class Docker(suiteConfig: Config = ConfigFactory.empty,
     val containerConfig: ContainerConfig = buildContainerConfig(nodeConfig, ip, restApiPort, networkPort)
     val containerId = Try {
       client.createContainer(containerConfig, configuredNodeName + "-" + hashString)
-    }.recoverWith { case e: ImageNotFoundException =>
-      Failure(new Exception(s"Error: docker image is missing ($e)\nRun 'sbt it:test' to generate it."))
+    }.recoverWith {
+      case e: ImageNotFoundException =>
+        Failure(new Exception(s"Error: docker image is missing ($e)\nRun 'sbt it:test' to generate it."))
```

Comment:
why don't you just pass it as a second parameter to the new Exception?

Reply:
Oh, sure. Sorry, I missed it the first time. Fixed.